src/app/dev/DevToys.Api/Tool/GUI/Components/UIHoverTooltip.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace DevToys.Api;
+
+public record UIHoverTooltip
+{
+    public string Word { get; }
```

Comment:
I wonder if this should be a `TextSpan` (so a position in the text document) instead of a word.

Reply:
Oko